src/flows/pipes/Notification/view.tsx

```diff
@@ -259,243 +255,21 @@ const Notification: FC<PipeProp> = ({Context}) => {
-}
-
-task_data = ${format_from_js_file(ast)}
-trigger = ${conditions}
-messageFn = (r) => ("${data.message}")
-
-${ENDPOINT_DEFINITIONS[data.endpoint]?.generateQuery(data.endpointData)}
-|> monitor["deadman"](t: experimental["subDuration"](from: now(), d: ${
-      deadman.deadmanCheckValue
-    }))`
-
-    const newAST = parse(newQuery)
-
-    if (!params.name) {
-      params.name = `"Notebook Deadman Task for ${id}"`
-    }
-
-    if (data.interval) {
-      params.every = data.interval
-    }
-
-    if (data.offset) {
-      params.offset = data.offset
-    }
-
-    if (Object.keys(vars).length) {
-      const varString = Object.entries(vars)
-        .map(([key, val]) => `${key}: ${val}`)
-        .join(',\n')
-      const varHeader = parse(`option v = {${varString}}\n`)
-      newAST.body.unshift(varHeader.body[0])
-    }
-
-    const paramString = Object.entries(params)
-      .map(([key, val]) => `${key}: ${val}`)
-      .join(',\n')
-    const taskHeader = parse(`option task = {${paramString}}\n`)
-    newAST.body.unshift(taskHeader.body[0])
-
-    return format_from_js_file(newAST)
-  }, [
-    id,
-    queryText,
-    data.every,
-    data.offset,
-    data.endpointData,
-    data.endpoint,
-    data.thresholds,
-    data.message,
-  ])
-
-  const generateTask = useCallback(() => {
-    if (data.thresholds[0].type === deadmanType) {
-      return generateDeadmanTask()
-    }
-    // simplify takes care of all the variable nonsense in the query
-    const ast = parse(simplify(queryText))
-
-    const vars = remove(
-      ast,
-      node => node.type === 'OptionStatement' && node.assignment.id.name === 'v'
-    ).reduce((acc, curr) => {
-      // eslint-disable-next-line no-extra-semi
-      ;(curr.assignment?.init?.properties || []).reduce((_acc, _curr) => {
-        if (_curr.key?.name && _curr.value?.location?.source) {
-          _acc[_curr.key.name] = _curr.value.location.source
-        }
-
-        return _acc
-      }, acc)
-
-      return acc
-    }, {})
-    const params = remove(
-      ast,
-      node =>
-        node.type === 'OptionStatement' && node.assignment.id.name === 'task'
-    ).reduce((acc, curr) => {
-      // eslint-disable-next-line no-extra-semi
-      ;(curr.assignment?.init?.properties || []).reduce((_acc, _curr) => {
-        if (_curr.key?.name && _curr.value?.location?.source) {
-          _acc[_curr.key.name] = _curr.value.location.source
-        }
-
-        return _acc
-      }, acc)
-
-      return acc
-    }, {})
-
-    const conditions = data.thresholds
-      .map(threshold => THRESHOLD_TYPES[threshold.type].condition(threshold))
-      .join(' and ')
-
-    const newQuery = `
-import "strings"
-import "regexp"
-import "influxdata/influxdb/monitor"
-import "influxdata/influxdb/schema"
-import "influxdata/influxdb/secrets"
-import "experimental"
-${ENDPOINT_DEFINITIONS[data.endpoint]?.generateImports()}
-
-check = {
-	_check_id: "${id}",
```

Comment:
seems like this formatting was wrong too 😬

Reply:
good catch. made the format correct.